client/js/components/common/canvas_authentication.jsx

```diff
@@ -7,14 +7,18 @@ const select = state => ({
 });
 
 export class CanvasAuthentication extends React.Component {
+  static propTypes = {
+    settings: React.PropTypes.object.isRequired,
+  };
 
-  renderSettings(){
-    return _.map(this.props.settings, (value, key) => {
-      return <input key={key} type="hidden" value={value} name={key} />
-    })
+  renderSettings() {
+    return _.map(
+      this.props.settings,
+      (value, key) => <input key={key} type="hidden" value={value} name={key} />
```

Comment:
Refactored to return a single expression. Complies with http://eslint.org/docs/rules/arrow-body-style.